### ISSUE TITLE
[BUG]: llama 7b finetuning

### ISSUE THREAD

**Issue by binmakeswell** [hpcaitech/ColossalAI#4867]
### 🐛 Describe the bug

[From the community]
llama 7b finetuning with the same parameters, gemini-auto fine-tuning produces a model that doesn't work and the loss goes to nan, gemini converges normally.
I changed the hyperparameters maxlength to 128 and left the rest untouched, and changed the dataset to my Chinese qa. 

### Environment

python3.8 torch1.13.1 cuda11.7
2 a800 GPU

**Comment by TongLi3701** [comment]
Whi'ch

**Comment by binmakeswell** [comment]
> Whi'ch

Wechat group
